hadoop-ozone/common/src/main/proto/OzoneManagerProtocol.proto

```diff
@@ -677,7 +677,7 @@ message ListBucketsResponse {
 message KeyArgs {
     required string volumeName = 1;
     required string bucketName = 2;
-    required string keyName = 3;
```

Comment:
Change required to option will be an incompatible change.

Reply:
I will change this place back to required and not change the KeyArgs,  by use list of KeyArgs to delete.